apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -871,7 +871,11 @@ async fn router(
             ingest_route(
                 req,
                 // without explicit version, go to current project version
-                route.join(current_version),
+                if !project.features.data_model_v2 {
```

Comment:
Can you explain why we need that flow control?

Reply:
I wonder if we should keep a version attribute to the object in DMV2, so that the user can have similar experiences but without DCM, ie they would be declaring versions explicitly but have a field for it so that it doesn't have to be in the name.

Reply:
Gonna merge this in so GL can test with python. I have the version thing we talked about on my todo list.